base_classes/NXparameters.nxdl.xml

```diff
@@ -27,7 +27,7 @@
     name="NXparameters" 
 	type="group" extends="NXobject">
     <doc>Container for parameters, usually used in processing or analysis.</doc>
-    <field name="term" minOccurs="0" maxOccurs="unbounded" type="NX_CHAR">
+    <field name="TERM" minOccurs="0" maxOccurs="unbounded" nameType="any">
```

Comment:
`type="NX_CHAR"` is removed which changes the definition?

Reply:
`nameType="any"` makes the capitalized `TERM` explicit. I did not know this. Otherwise from an NXDL pov the name "TERM" would be a fixed name.

Perhaps we should do something in the HTML rendering to make this explicit because I thought we just relied on all-caps to know whether it is a fixed name or not.

Reply:
The default data type for all NeXus fields is `NX_CHAR`, so nothing is changed.  It was not necessary to specify it here.